grpc/health/v1/health.proto

```diff
@@ -39,6 +39,14 @@ message HealthCheckResponse {
   ServingStatus status = 1;
 }
 
+message HealthListRequest {
+  repeated string services = 1; // Contains the name of the services that should be obtained. If empty, must returns all.
+}
+
+message HealthListResponse {
+  map<string, HealthCheckResponse> services_statuses = 2; // Contains all the service and their respective status.
```

Comment:
I find it strange if there's only one field and it starts with ID 2. Technically fine, just seems like a strange / uninentional choice. I also prefer the shorter field name.
```suggestion
  map<string, HealthCheckResponse> statuses = 1; // Contains all the services and their respective status.
```

Reply:
You're right on all accounts, thanks for the review!